schemas/chemicalSubstance.schema.tpl.json

```diff
@@ -6,20 +6,20 @@
   "properties": {
     "additionalRemarks": {
       "type": "string",
-      "_instruction": "Enter additional remarks about the chemical substance (pure or within a mixture)."
+      "_instruction": "Enter any additional remarks concering this chemical substance."
     },
     "lookupLabel": {
       "type": "string",
-      "_instruction": "Add an appropriate look up label for this substance."
+      "_instruction": "Enter a lookup label for this chemical substance that may help you to find back to this instance more easily."
```

Comment:
You can't say "find back to". "get back to" is a possibility. I suggest either:

"...that may help you to find this instance more easily."

or

""...that may help you to find this instance again."

Reply:
Thanks! I used "...that may help you to find this instance more easily."